StDb/idl/dummy.idl

```diff
@@ -0,0 +1,10 @@
+/*    dummy.idl
+*
+*  Table: dummy
+*
+*       description: //  dummy file
+*/
+
+struct dummy {.
```

Comment:
Is there an extra dot at the end of line?

Reply:
That dot does not matter, it is not a C++ file. But, it clearly demonstrates why reviews can delay the merge, right? As I mentioned in the title, submitted IDL file is compilable by cons and it produces valid .h and .inc files. Create an "StDb/idl" directory at rcas, place this file inside, run cons, see it producing a library. This should be done by CI script, really.

Reply:
Yes, a PR with CI scripts is very welcome. Let me know if you want to work on it.

Reply:
Well, SPHENIX has CI governed by Jenkins: https://web.sdcc.bnl.gov/jenkins-sphenix/
STAR Infrastructure team may consult Chris and SDCC folks about CI setup details. I'd love to do that myself (CI is great), but I have plenty of tasks on my plate already, and this list will grow rapidly when RHIC Run 21 will be over.

Reply:
If you don't have time like most of us then we'll just use what we have.
Now, you see how useful this PR was. Now I know that idl files are not in C++ and taht having a dot at the end of line is OK. You shared the knowledge. Thank you